hilti/runtime/include/hilti.h

```diff
@@ -28,7 +28,7 @@ void print(const T& t, bool newline = true) {
     cout << hilti::rt::to_string_for_print(t);
 
     if ( newline )
-        cout << std::endl;
+        cout << '\n';
```

Comment:
Note there's semantic difference: `endl` flushes, which for buffered `cout` can make a difference. It's unlikely to matter here, so seems fine, but I am a bit surprised that clang-tidy is generally suggesting that change,

Reply:
Yepp, that's exactly [the reason this lint gives](https://clang.llvm.org/extra/clang-tidy/checks/performance/avoid-endl.html). For our code there was only a single place where we wanted the flush which I [now made explicit](https://github.com/zeek/spicy/blob/b763e7c56f99512c429724ea15ed7b72d384b908/hilti/runtime/src/debug-logger.cc#L47).